daemon/src/process_manager.rs

```diff
@@ -72,7 +72,7 @@ impl Actor {
 
         // 2. Post process event
         use EventKind::*;
-        match event.event {
+        match &event.event {
```

Comment:
❓ Why is all the subsequent cloning preferable over matching by value here?

Reply:
- I needed the full value to dispatch it to projection.
- If I clone the entire event here I also clone all of the CETs which I never need by value in the match.